verbal/20160530/main.tex

```diff
@@ -0,0 +1,68 @@
+\documentclass[11pt]{meetingmins}
+\usepackage[utf8]{inputenc}
+
+\setcommittee{BugBusters}
+
+\setmembers{
+
+  D.~Polonio,
+  D.~Rigoni,
+  M.~Di Pirro,
+  G.~Mazzocchin,
+  L.~Bianco,
+  A.~Mantovani,
+  E.~Carraro
+}
+
+\setdate{May 24, 2016}
+
+\setpresent{
+  
+  D.~Polonio,
+  E.~Carraro,
+}
+
+\absent{
+  D.~Rigoni,
+  A.~Mantovani,
+  L.~Bianco,
+  M.~Di Pirro,
+  G.~Mazzocchin
+}
+
+\alsopresent{None}
+
+\begin{document}
+
+\maketitle
+
+\section{Ordine del giorno}
+
+\begin{enumerate}
+
+\item Decidere il formato migliore per il manuale sviluppatore
+\item Spiegazione del Workflow per Heroku
+\end{enumerate}
+
+\section{Orari}
+
+\begin{itemize}
+\item[Inizio]: ore 10:38
+\item[Fine]: ore 11:00
+\item[Tempo]: Previsto di 20 min, effettivo di 15 min
+
+\end{itemize}
+
+\section{Verbale}
+
+\subsubsection{Decidere il formato migliore per il manuale sviluppatore}
+Viene deciso dai presenti che per rendere più facile la consultazione del documento verrà creata una pagina di presentazione. Bisognerà quindi creare uno script per poter generare automaticamente ogni volta che viene invocato il comando ``npm run doc'' generi anche codesta pagina.
```

Comment:
Nel senso che sopra la cartella `api/` e `source/` ci sia un `index.html`, che semplicemente punti alla homepage della cartella `api` e alla homepage della cartella `souce/`. In questa maniera, quando consegneremo la documentazione a tullio, non si potra' lamentare della difficile documentazione come ha fatto nella consegna delle RQ :blush: 
L'index in poche parole ha lo stesso stile di MaaS e all'interno ha i due link, nient'altro. :blush: